lib/std/math/ceil.zig

```diff
@@ -62,6 +62,8 @@ fn ceil32(x: f32) f32 {
 }
 
 fn ceil64(x: f64) f64 {
+    const f64_toint = 1.0 / math.floatEps(f64);
```

Comment:
There is no commit message, which explains that `ceil64` was broken and how it got fixed.

Reply:
It was not broken, and it did not get fixed. The `fXX_toint` constants are getting deprecated (see #10133) as they have a very confusing name and are literally only used in the rounding functions and nowhere else, and it's just `1 / epsilon`. This is part of the cleanup.

Reply:
ah, my bad then.